SECURITY.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "Security Policy"
+linkTitle: "Security policy"
+weight: 10
+type: "docs"
+aliases:
+  - /contributing/security-policy/
+---
+
+# Introduction
+
+Security is core to our values, and we value the input of developers, operators,
+security researchers, and others acting in good faith to help us maintain a high
+standard for the security and privacy for our users, and their users. This
+includes encouraging responsible vulnerability research and disclosure. This
+policy sets out our definition of good faith in the context of finding and
+reporting vulnerabilities, as well as what you can expect from us in return.
+
+# Expectations
+
+When working with us according to this policy, you can expect us to:
+
+- Extend Safe Harbor for your vulnerability research that is related to this
+  policy;
+- Work with you to understand and validate your report, including a timely
+  initial response to the submission;
+- Work to remediate discovered vulnerabilities in a timely manner;
+- Publicly disclose accepted vulnerabilities in a timely manner; and
+- Recognize your contribution to improving our security if you are the first to
+  report a unique vulnerability, and your report triggers a code or
+  configuration change.
+
+TODO: Is Knative Security up for including the recognize clause? Do we have a
+page for this?
+
+# Ground Rules
+
+To encourage vulnerability research and to avoid any confusion between
+good-faith hacking and malicious attack, we ask that you:
+
+- Play by the rules. This includes following this policy, as well as any other
+  relevant agreements. If there is any inconsistency between this policy and any
+  other relevant terms, the terms of this policy will prevail;
```

Comment:
I would put CoC above this.

Reply:
added as one of the relevant agreements.

Reply:
Added as a relevant agreement.